static/js/components/CourseList.js

```diff
@@ -29,37 +43,75 @@ class CourseList extends React.Component {
           coursesPassed++;
         }
 
-        return <ul
+        let courseRuns = [];
+        if (expander[course.id]) {
+          courseRuns = course.runs.map(run => {
+            return <Grid style={{height: DASHBOARD_RUN_HEIGHT}} key={run.id}>
+              <Cell
+                col={6}
+                className="course-run-title"
+              >
+                {run.title}
+              </Cell>
+              <Cell col={4} className="course-run-status">
+                {makeRunStatusDisplay(run)}
+              </Cell>
+            </Grid>;
+          });
+        }
+
+        let expanderClass = "glyphicon glyphicon-chevron-down";
+        if (expander[course.id]) {
+          expanderClass = "glyphicon glyphicon-chevron-up";
+        }
+
+        let height = DASHBOARD_COURSE_HEIGHT + courseRuns.length * DASHBOARD_RUN_HEIGHT;
+        return <div
           key={course.id}
-          className={"course-list-body course-list-status-" + course.status}
+          className={"course-list-body course-list-row course-list-status-" + course.status}
```

Comment:
Looks like a good idea, I'll file an issue to deal with this separately since there's a lot of code to change